internal/service/emr/managed_scaling_policy_test.go

```diff
@@ -15,12 +15,24 @@ import (
 	tfemr "github.com/hashicorp/terraform-provider-aws/internal/service/emr"
 )
 
+func init() {
+	acctest.RegisterServiceErrorCheckFunc(emr.EndpointsID, testAccErrorCheckSkipEMR)
+}
+
+func testAccErrorCheckSkipEMR(t *testing.T) resource.ErrorCheckFunc {
+	return acctest.ErrorCheckSkipMessagesContaining(t,
+		"Managed scaling is not available",
+		"SSO is not enabled",
+		"Account is not whitelisted to use this feature",
```

Comment:
AWS has deprecated the use of the terms "whitelist" and "blacklist" in favor of "allow-list" and "deny-list".